BitwardenShared/UI/Auth/Landing/LandingProcessor.swift

```diff
@@ -22,6 +23,9 @@ class LandingProcessor: StateProcessor<LandingState, LandingAction, LandingEffec
     /// The services required by this processor.
     private let services: Services
 
+    /// Helper class with region specific functions
+    var regionHelper: RegionHelper?
```

Comment:
🎨 Would it work to make this private and initialize it in the processor (and in `StartRegistrationProcessor`)? It doesn't look like we need to mock this directly so the coordinator doesn't necessarily need to set it on the processor. Then it wouldn't be possible for it to be `nil` or missed from being initialized.

Because of the delegate, it would need to be lazy initialized:
```
    private lazy var regionHelper = RegionHelper(
        coordinator: coordinator,
        delegate: self,
        stateService: services.stateService
    )
```

Reply:
Yes that worked out thanks!